mongodb-panache-quickstart/src/test/java/org/acme/mongodb/panache/PersonResourceTest.java

```diff
@@ -0,0 +1,217 @@
+import de.flapdoodle.embed.mongo.MongoImportStarter;
+import de.flapdoodle.embed.mongo.MongodExecutable;
+import de.flapdoodle.embed.mongo.MongodStarter;
+import de.flapdoodle.embed.mongo.config.IMongoImportConfig;
+import de.flapdoodle.embed.mongo.config.IMongodConfig;
+import de.flapdoodle.embed.mongo.config.MongoImportConfigBuilder;
+import de.flapdoodle.embed.mongo.config.MongodConfigBuilder;
+import de.flapdoodle.embed.mongo.config.Net;
+import de.flapdoodle.embed.mongo.config.RuntimeConfigBuilder;
+import de.flapdoodle.embed.mongo.distribution.Version;
+import de.flapdoodle.embed.process.config.IRuntimeConfig;
+import de.flapdoodle.embed.process.runtime.Network;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import io.restassured.config.ObjectMapperConfig;
+import io.restassured.http.ContentType;
+import io.restassured.parsing.Parser;
+import org.assertj.core.api.Assertions;
+import org.bson.types.ObjectId;
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.Objects;
+
+import static io.restassured.config.LogConfig.logConfig;
+
+@QuarkusTest
+class PersonResourceTest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(PersonResourceTest.class);
+    private static MongodExecutable MONGO;
+
+    @BeforeAll
+    static void initAll() throws IOException {
+        final int mongoPort = 27017;
+        LOGGER.info("Starting Mongo {} on port {}", Version.Main.V4_0, mongoPort);
+        Net net = new Net(mongoPort, Network.localhostIsIPv6());
+        IMongodConfig mongodConfig = new MongodConfigBuilder().version(Version.Main.V4_0).net(net).build();
+        IRuntimeConfig runtimeConfig = new RuntimeConfigBuilder().defaults(Command.MongoD).build();
+        MONGO = MongodStarter.getInstance(runtimeConfig).prepare(mongodConfig);
+        MONGO.start();
+
+        File jsonFile = new File(Objects
+                .requireNonNull(Thread.currentThread().getContextClassLoader().getResource("person-dataset.json")).getFile());
+        String importDatabase = "person";
+        String importCollection = "ThePerson";
+        MongoImportExecutable mongoImportProcess = mongoImportExecutable(mongoPort, importDatabase,
+                importCollection, jsonFile.getAbsolutePath(), true, true, true);
+        String mongoUrl = ConfigProvider.getConfig().getValue("quarkus.mongodb.connection-string", String.class);
+        MongoClient mongoClient = MongoClients.create(mongoUrl);
+        mongoImportProcess.start();
+        Assertions.assertThat(mongoClient.getDatabase(importDatabase).getCollection(importCollection).estimatedDocumentCount())
+                .isEqualTo(3);
+        mongoImportProcess.stop();
```

Comment:
This seems overly complicated to start MongoDB, keeps in minds that the Quickstart is intended to be easily readable for people how wants to understand how to quicly start working with Quarkus and a specific extension.
Please stick to the same startup code as the integration test, I understand that you are using some JSON init sript, better switch to plain POST using Restassured.